src/helm/benchmark/scenarios/unitxt_scenario.py

```diff
@@ -0,0 +1,66 @@
+from typing import List
+from datasets import load_dataset
+
+from helm.benchmark.scenarios.scenario import (
+    Output,
+    Reference,
+    Scenario,
+    Instance,
+    Input,
+    CORRECT_TAG,
+    TRAIN_SPLIT,
+    TEST_SPLIT,
+    VALID_SPLIT,
+)
+
+
+class UnitxtScenario(Scenario):
+    """Integration with https://www.unitxt.org/"""
+
+    name = "unitxt"
+    description = "Unitxt Scenarios"
+    tags = ["unitxt"]
+
+    UNITXT_SPLIT_NAME_TO_HELM_SPLIT_NAME = {
+        "train": TRAIN_SPLIT,
+        "test": TEST_SPLIT,
+        "validation": VALID_SPLIT,
+    }
+
+    def __init__(self, **kwargs):
+        super().__init__()
+        self.kwargs = kwargs
+
+    def get_instances(self, output_path: str) -> List[Instance]:
+        dataset_name = ",".join(f"{key}={value}" for key, value in self.kwargs.items())
+        dataset = load_dataset("unitxt/data", dataset_name, trust_remote_code=True)
+
+        instances: List[Instance] = []
+
+        for unitxt_split_name, helm_split_name in UnitxtScenario.UNITXT_SPLIT_NAME_TO_HELM_SPLIT_NAME.items():
+            for index, row in enumerate(dataset[unitxt_split_name]):
+                correct_reference = row["target"]
+                references = [
+                    Reference(
+                        output=Output(text=reference_text),
+                        tags=[CORRECT_TAG] if reference_text == correct_reference else [],
```

Comment:
All elements  in the "references" list are correct.  Target is just an arbitrary one that is used for few shot demonstrations.   So perhaps CORRECT_TAG should be marked for all?

Reply:
Changed.